packages/config/config/devices/0x0438/touch-thermostat-16a.json

```diff
@@ -0,0 +1,348 @@
+{
+	"manufacturer": "Namron",
+	"manufacturerId": "0x0438",
+	"label": "4512745",
+	"description": "In-Wall Touch Thermostat 16A",
+	"devices": [
+		{
+			"productType": "0x0200",
+			"productId": "0xd025"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "2",
+			"label": "Over-Current Threshold",
+			"description": "When detected current is above threshold, relay turns off and sends notification. Allowable range: 0, 5-16",
```

Comment:
I'm guessing `0` means disable the function? We usually add a single option for these.